src/NHSD.BuyingCatalogue.Identity.Api/Controllers/OrganisationsController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Identity.Api.Models;
+using NHSD.BuyingCatalogue.Identity.Api.Repositories;
+using NHSD.BuyingCatalogue.Identity.Api.ViewModels.Organisations;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Controllers
+{
+    [Route("api/v1/Organisations")]
+    [ApiController]
+    [Produces("application/json")]
+    [AllowAnonymous]
+    public sealed class OrganisationsController : ControllerBase
+    {
+        private readonly IOrganisationRepository _organisationRepository;
+
+        public OrganisationsController(IOrganisationRepository organisationRepository)
+        {
+            _organisationRepository = organisationRepository;
+        }
+
+        [HttpGet]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult> GetAll()
+        {
+            IEnumerable<Organisation> organisationsList = await _organisationRepository.ListOrganisationsAsync();
+
+            return Ok(new GetAllOrganisationsViewModel
```

Comment:
Why have this view model?
IMHO, this should be Task<IEnumerable<OrganisationViewModel>>, as this is the simple case as described in the MS docs - https://docs.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-3.1

Reply:
GetAllOrganisationsViewModel is needed to fufil the JSON contract

Reply:
GetAllOrganisationsViewModel is needed to fulfill the JSON contract below

https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_sprints/taskboard/Buying%20Catalogue%20Team/Buying%20Catalogue/Iteration%2019?workitem=5201

Reply:
Hmm. Normally you'd just return a list of objects, not add the extra layer. I guess it's in the spec though, so fine.